tests/dataimportcron_test.go

```diff
@@ -784,6 +784,7 @@ var _ = Describe("DataImportCron", Serial, func() {
 				},
 			}
 			Expect(dataSource.Spec.Source).To(Equal(expectedSource))
+			waitForConditions(corev1.ConditionFalse, corev1.ConditionTrue)
```

Comment:
This just waits for the dataimportcron in the test to converge https://github.com/kubevirt/containerized-data-importer/blob/898e8946cc3221e927ce4b56783da756b254bcd6/tests/dataimportcron_test.go#L115-L119 Are you sure this is what you were after?

Reply:
I want to make sure that **all** dataimportcrons converged at the very end of the run. But honestly not sure if that's what my PR does :) will it work this way?

Reply:
In that case you want to do that in the AfterEach
something like LIST all dataimportcrons, iterate over each one and ensure `waitForConditions(corev1.ConditionFalse, corev1.ConditionTrue)`

Reply:
Okay, thanks! Giving it another try